misc/pytorch_toolkit/chest_xray_screening_federated_gcn/configs/fl_with_gnn.json

```diff
@@ -0,0 +1,38 @@
+{
+    "train": {
+        "data": "/storage/adityak/chest_x_ray/",
```

Comment:
Replace the path with the relative path. Check the repo  I shared as a reference.